be/src/storage/lake/lake_primary_index.cpp

```diff
@@ -28,6 +29,13 @@ namespace starrocks::lake {
 
 static bvar::LatencyRecorder g_load_pk_index_latency("lake_load_pk_index");
 
+LakePrimaryIndex::~LakePrimaryIndex() {
+    if (!_enable_persistent_index && _persistent_index != nullptr) {
+        auto st = LocalPkIndexManager::clear_persistent_index(_tablet_id);
+        LOG_IF(WARNING, !st.ok()) << "Fail to clear pk index from local disk: " << st.to_string();
```

Comment:
Just a frindly remind that you can write `LOG(level) << st` directly without calling `st.to_string()`